src/utils/custom-scroll.js

```diff
@@ -0,0 +1,32 @@
+import { makeStyles } from  './make-styles'
+
+/**
+ * @param {string} width scrollbar width, default = '0.125rem'. 
```

Comment:
Think we agreed to make it `3px` instead of `2px`. So that would be `0.1875rem`. Right now with 2px it's too thin.

Reply:
trueeeee. Sure! I will add the hover styling

Reply:
:( hover can only modify it's color

Reply:
:( in that case let's just darken it on hover. Darken both the scroll-area bg color and scrollbar color: 
- hoverThumbColor: `#AAAAAA` 
- hoverTrackColor: `#EAEAEA`

Reply:
:) also it's best to use our globally defined colors from tailwind config file (e.g. `secondary-400` + `secondary-200` and `secondary-500` + `secondary-300` for hover) instead of hex code. In case we end up changing our secondary color scheme one day and then we'll have to come to fix this manually

Reply:
It is very restricted for styling pseudo elements. I added the option of passing one of our color string which it automatically defines 400, 200 and 500, 300 for hover. Let me know what do you think